Gemfile

```diff
@@ -48,3 +47,7 @@ group :test do
   gem 'timecop', '0.8.1'
   gem 'govuk-content-schema-test-helpers', '~> 1.4.0'
 end
+
+# Upgrade to Sentry
+gem "govuk_app_config", "~> 0.2.0"
+gem "govuk_sidekiq", "~> 2.0.0"
```

Comment:
This app doesn't use Sidekiq, so it doesn't have to be added.

Reply:
👍